packages/loopring_v3/contracts/core/impl/ExchangeV3.sol

```diff
@@ -662,6 +647,40 @@ contract ExchangeV3 is IExchangeV3, ReentrancyGuard
     }
 
     // -- Admins --
+    function withdrawExchangeFees(
+        address token,
+        address recipient
+        )
+        external
+        override
+        nonReentrant
+        onlyOwner
+    {
+        require(recipient != address(0), "INVALID_ADDRESS");
+
+        uint amount = token.selfBalance();
+        token.transferOut(recipient, amount);
+    }
+
+    function withdrawUnregisteredToken(
+        address token,
+        address to,
+        uint    amount
+        )
+        external
+        override
+        nonReentrant
+        onlyOwner
+    {
+        try this.getTokenID(token) {
+            revert("TOKEN_REGISTERED");
+        } catch {
+            require(to != address(0), "INVALID_ADDRESS");
+            require(amount > 0, "INVALID_AMOUNT");
+            state.depositContract.transfer(address(0), to, token, amount);
```

Comment:
@Brechtpd shall I change `address(0)` to `address(state.depositContract)`?

Reply:
Instead of using `transfer` I think it's best to just use `withdraw` so we don't have to worry about token approvals:
```
state.depositContract.withdraw(address(state.depositContract), to, token, amount, new bytes(0));
```